docs/man/rpmbuild-config.5.scd

```diff
@@ -0,0 +1,304 @@
+RPMBUILD-CONFIG(5)
+
+# NAME
+*rpmbuild-config* - rpmbuild macro configuration
+
+# SYNOPSIS
+_NAME_ _VALUE_
+
+# DESCRIPTION
+The primary configuration mechanism in *rpmbuild* is via macros.
+See *rpm-config*(5) for the description of the general mechanism,
+this manual only describes the configurables affecting *rpmbuild*
+operation.
+
+# CONFIGURATION
+The following configurables are supported for the *rpm* build operation,
+typically invoked through the *rpmbuild* command.
+
+## Locations and filenames
+
+*%\_builddir* _DIRECTORY_
+	The directory where the per-package build directories will be placed
+	when building.
+
+*%\_fileattrsdir* _DIRECTORY_
+	The directory where the file classifier configuration is located.
+
+*%\_rpmdir* _DIRECTORY_
+	The directory where newly built binary packages will be written.
+
+*%\_rpmfilename* _TEMPLATE_
+	The output binary package filename template used when building
+	binary packages. The template is macro expanded, and the result
+	is considered a filename relative to *%\_rpmdir*.
+
+*%\_sourcedir* _DIRECTORY_
+	The directory where the sources and patches are found when building.
+
+*%\_srcrpmdir* _DIRECTORY_
+	The directory where newly built source packages will be written.
+
+*%\_topdir* _DIRECTORY_
+	The toplevel directory under which the package build directories
+	reside by default.
+
+## Informational tags
+These are a special group of macros that if defined, behave as if the
+corresponding tag had been used in a specfile. This allows changing
+the values without modifying thousands of specfiles.
+
+*%bugurl* _URL_
+	Configurable bug URL, same as BugURL: tag in a specfile.
+	If set, should point to a website for searching for existing
+	and filing new issues for this package.
+
+*%distribution* _STRING_
+	Configurable distribution informationsame as Distribution: tag in a
+	specfile.
+
+*%disturl* _URL_
+	Configurable distribution URL, same as DistURL: tag in a specfile.
+
+*%disttag* _STRING_
+	Configurable distribution tag, same as DistTag: tag in a specfile.
+
+*%modularitylabel* _STRING_
+	Configurable modularity label, same as ModularityLabel: in a specfile.
+	Obsolete.
+
+*%packager* _STRING_
+	Configurable packager information, same as Packager: in a specfile.
+
+*%translationurl* _URL_
+	Configurable translation URL, same as TranslationURL: in a specfile.
+
+*%upstreamreleases* _URL_
+	Configurable URL for checking for newer upstream releases, same as
+	UpstreamReleases: in a specfile.
+
+*%vendor* _STRING_
+	Configurable vendor information, same as Vendor: in a specfile.
+
+## Build operation
+These settings affect various aspects of the build, but have no direct
+impact on the produced packages.
```

Comment:
May be add "but may make builds fail or succeed. These failures may be desired for failed checks or not when running out of resources" or something along this line.